react-native/react/common-adapters/index.desktop.js

```diff
@@ -5,3 +5,5 @@ export {default as Input} from './input'
 export {default as Header} from './header'
 export {default as Logo} from './logo'
 export {default as Text} from './text'
+export {default as Terminal} from './terminal'
+export {default as Icon} from './icon'
```

Comment:
Add these to `index.js.flow` (If you were importing these from `index` instead of `index.desktop` flow would give you this error)